### PR TITLE
Add Certification Guidelines to site

### PR DESCRIPTION
**Pull request by ScottRudiger**
This PR makes our certification guidelines publically available so we can share a link with developers rather than sending the document each time.

### CODE REVIEW

File: source/integrations/certification-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+### End-to-End Sales Tax Automation
+
+Both calculations and transaction push will be required. For a great TaxJar experience, a merchant has to be able to calculate sales tax and remit it later using AutoFile.
+
+- Sales tax calculations
+- Sales tax reporting via order / refund push
+- Easy merchant onboarding and authentication
+- Easy configuration panel
+ - Enable / disable calculations
```
Comment (fastdivision) [inline source/integrations/certification-guidelines.md]: Let's clean up the whitespace before these list items

File: source/integrations/certification-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Product taxability and exemption support
+- Transaction backfill via API or CSV export
+
+We also want to ensure certified integrations only make API calculations when absolutely necessary to save merchants money:
+
+- API call conservation
+ - Only make calculations when a merchant has nexus; send in all transactions
+ - Cache API calls until order details change
+
+Certified integrations should leverage their TaxJar partnership effectively and include the necessary code to track API usage across one or more API tokens. Referral links should be set up properly to track commission in PartnerStack (formerly known as GrowSumo). Later, the `plugin` param will be used for transactions shown in the TaxJar app to differentiate them from other API transactions:
```
Comment (fastdivision) [inline source/integrations/certification-guidelines.md]: Let's remove the reference to specific referral PRM tools here:
```
Referral links should be set up properly to track commission.
```

File: source/integrations/certification-guidelines.md (hunk 1 above)
Comment (ScottRudiger) [reply source/integrations/certification-guidelines.md]: @fastdivision this is how the sub-bullet points are being indented. Would you rather these ones not be indented?

**Comment by fastdivision** [comment]
Looking good. Can you look into tweaking the line height / spacing for the nested lists? There should be a bit more spacing above "Enable / disable calculations":

Screen Shot 2019-10-29 at 8 41 17 PM

**Comment by ScottRudiger** [comment]
@fastdivision I think the spacing looks more consistent now.

**Pull request merged**